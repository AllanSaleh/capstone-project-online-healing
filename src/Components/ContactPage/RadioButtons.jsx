import React from 'react';

import ContactPic from './Images/ContactPic.svg';

export default function RadioButtons() {
  const radios = [
    { value: 'ServiceQuestion', text: 'I have a question about the service.' },
    {
      value: 'ClientSupport',
      text: "I'm a registered client and I need support.",
    },
    {
      value: 'CouncelorInterest',
      text: "I'm a counselor interested in joining.",
    },
    {
      value: 'CounselorSupport',
      text: "I'm a counselor interested in joining.",
    },
    {
      value: 'BusinessInquiry',
      text: "I'm a registered counselor and I need support.",
    },
    {
      value: 'OrganizationInterest',
      text: "I'm interested in Healing Online for my organization.",
    },
    { value: 'BillingQuestion', text: 'I have a billing related question.' },
  ];
  return (
    <div className="h-firstsection pt-navbar px-sides flex flex-col justify-evenly w-full">
      <div>
        <div className="text-2xl sm:text-title sm:mb-0 mb-3">
          SEND US YOUR REQUEST!
        </div>
        <div className="sm:text-subtitle sm:mb-0 mb-10  text-gray-700 md:text-sm lg:text-xl lg:text-gray-700 lg:text-subtitle md:mt-5 md:flex-col md:justify-evenly md:flex">
          Do you have a question, concern, idea, feedback, or problem? If you
          need assistance, please fill out the form below and we'd be happy to
          help!
        </div>
      </div>

      <div className="flex justify-between sm:flex-row flex-col">
        <div>
          <div className="text-subtitle mb-4 text-sm sm:text-lg">Type of Contact</div>
          <div className="flex flex-col justify-between">
            {radios.map((radio) => (
              <div className="flex ml-4 my-2 ">
                <input
                  name="ContactRadio"
                  className="my-2 mr-4"
                  type="radio"
                  value={radio.vlaue}
                />
                <div className="text-paragraph sm:text-lg text-xm">{radio.text}</div>
              </div>
            ))}
          </div>
        </div>
        <img src={ContactPic} className="sm:block hidden" alt="contact pic" />
      </div>
    </div>
  );
}
