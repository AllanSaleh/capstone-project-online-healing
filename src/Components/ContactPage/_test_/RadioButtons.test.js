import * as React from 'react';
import * as renderer from 'react-test-renderer';
import RadioButtons from '../RadioButtons';

it('Radio Buttons renders correctly', () => {
  const tree = renderer.create(<RadioButtons />).toJSON();
  expect(tree).toMatchSnapshot();
});
