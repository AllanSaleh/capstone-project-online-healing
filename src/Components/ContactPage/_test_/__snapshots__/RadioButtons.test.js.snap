// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio Buttons renders correctly 1`] = `
<div
  className="h-firstsection pt-navbar px-sides flex flex-col justify-evenly w-full"
>
  <div>
    <div
      className="text-2xl sm:text-title sm:mb-0 mb-3"
    >
      SEND US YOUR REQUEST!
    </div>
    <div
      className="sm:text-subtitle sm:mb-0 mb-10  text-gray-700 md:text-sm lg:text-xl lg:text-gray-700 lg:text-subtitle md:mt-5 md:flex-col md:justify-evenly md:flex"
    >
      Do you have a question, concern, idea, feedback, or problem? If you need assistance, please fill out the form below and we'd be happy to help!
    </div>
  </div>
  <div
    className="flex justify-between sm:flex-row flex-col"
  >
    <div>
      <div
        className="text-subtitle mb-4 text-sm sm:text-lg"
      >
        Type of Contact
      </div>
      <div
        className="flex flex-col justify-between"
      >
        <div
          className="flex ml-4 my-2"
        >
          <input
            className="my-2 mr-4"
            name="ContactRadio"
            onChange={[Function]}
            type="radio"
            value="ServiceQuestion"
          />
          <div
            className="text-paragraph sm:text-lg text-xm"
          >
            I have a question about the service.
          </div>
        </div>
        <div
          className="flex ml-4 my-2"
        >
          <input
            className="my-2 mr-4"
            name="ContactRadio"
            onChange={[Function]}
            type="radio"
            value="ClientSupport"
          />
          <div
            className="text-paragraph sm:text-lg text-xm"
          >
            I'm a registered client and I need support.
          </div>
        </div>
        <div
          className="flex ml-4 my-2"
        >
          <input
            className="my-2 mr-4"
            name="ContactRadio"
            onChange={[Function]}
            type="radio"
            value="CouncelorInterest"
          />
          <div
            className="text-paragraph sm:text-lg text-xm"
          >
            I'm a counselor interested in joining.
          </div>
        </div>
        <div
          className="flex ml-4 my-2"
        >
          <input
            className="my-2 mr-4"
            name="ContactRadio"
            onChange={[Function]}
            type="radio"
            value="CounselorSupport"
          />
          <div
            className="text-paragraph sm:text-lg text-xm"
          >
            I'm a counselor interested in joining.
          </div>
        </div>
        <div
          className="flex ml-4 my-2"
        >
          <input
            className="my-2 mr-4"
            name="ContactRadio"
            onChange={[Function]}
            type="radio"
            value="BusinessInquiry"
          />
          <div
            className="text-paragraph sm:text-lg text-xm"
          >
            I'm a registered counselor and I need support.
          </div>
        </div>
        <div
          className="flex ml-4 my-2"
        >
          <input
            className="my-2 mr-4"
            name="ContactRadio"
            onChange={[Function]}
            type="radio"
            value="OrganizationInterest"
          />
          <div
            className="text-paragraph sm:text-lg text-xm"
          >
            I'm interested in Healing Online for my organization.
          </div>
        </div>
        <div
          className="flex ml-4 my-2"
        >
          <input
            className="my-2 mr-4"
            name="ContactRadio"
            onChange={[Function]}
            type="radio"
            value="BillingQuestion"
          />
          <div
            className="text-paragraph sm:text-lg text-xm"
          >
            I have a billing related question.
          </div>
        </div>
      </div>
    </div>
    <img
      alt="contact pic"
      className="sm:block hidden"
      src="ContactPic.svg"
    />
  </div>
</div>
`;
